src/components/filterType/FilterType.js

```diff
@@ -3,20 +3,16 @@ import React from 'react';
 // css
 import './FilterType.css'
 
-function FilterType({ heading, inputs, setFilters }) {
+function FilterType({ heading, inputs, setFilters, isChecked, setIsChecked }) {
     const updateCategory = (filters, input) => {
```

Comment:
Could use a more readable name. By input, it's difficult to understand what kind of value it holds.

Reply:
will update